builtin/providers/aws/resource_aws_inspector_assessment_target.go

```diff
@@ -0,0 +1,117 @@
+package aws
+
+import (
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/inspector"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAWSInspectorAssessmentTarget() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsInspectorAssessmentTargetCreate,
+		Read:   resourceAwsInspectorAssessmentTargetRead,
+		Update: resourceAwsInspectorAssessmentTargetUpdate,
+		Delete: resourceAwsInspectorAssessmentTargetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+			"arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"resource_group_arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceAwsInspectorAssessmentTargetCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).inspectorconn
+
+	targetName := d.Get("name").(string)
+	resourceGroupArn := d.Get("resource_group_arn").(string)
```

Comment:
I have changed the field to required, so now it should fail validation on plan if resource group isn't set.